Content.Server/_Shitmed/Medical/Surgery/SurgerySystem.cs

```diff
@@ -200,7 +200,7 @@ public FixedPoint2 DamageToBeDealt(Entity<SurgeryContaminableComponent> ent, Fix
         var exceedsAmount = (dirtiness - ent.Comp.DirtinessThreshold).Float();
         var additionalDamage = (1f / ent.Comp.InverseDamageCoefficient.Float()) * (exceedsAmount * exceedsAmount);
 
-        return FixedPoint2.New(additionalDamage) + ent.Comp.BaseDamage;
+        return FixedPoint2.Min(FixedPoint2.New(additionalDamage) + ent.Comp.BaseDamage, ent.Comp.ToxinStepLimit);
```

Comment:
comment

Reply:
isn't this already within the deltav comment block

Reply:
true